### PR TITLE
Feature/cli

### PR DESCRIPTION
**Pull request by iruzevic**
# Description

- corrections to WP cli init method to use new service interface
- fixes to command to create custom WP cli command

### CODE REVIEW

File: tests/WpCli/WpCliExampleTest.php [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+// 	public function getCommandName(): string
+// 	{
+// 		return 'command_name';
+// 	}
+
+// 	public function getDoc(): array
+// 	{
+// 		return [];
+// 	}
+// };
```
Comment (dingo-d) [inline tests/WpCli/WpCliExampleTest.php:22]: Remove commented code

File: tests/WpCli/WpCliTest.php [hunk 2]
```diff
@@ -37,11 +38,12 @@
 	$customCommand([], $customCommand->getDevelopArgs([]));
 
 	// Check the output dir if the generated method is correctly generated.
-	$customCommand = \file_get_contents(\dirname(__FILE__, 3) . '/cliOutput/src/CliCommands/TestCustomCommand.php');
+	$customCommand = \file_get_contents(\dirname(__FILE__, 3) . '/cliOutput/src/WpCli/TestWpCli.php');
 
-	$this->assertStringContainsString('class TestCustomCommand extends AbstractCustomCommand', $customCommand);
-	$this->assertStringContainsString('function getCommandName', $customCommand);
-	$this->assertStringContainsString('function getDoc', $customCommand);
+	$this->assertStringContainsString('class TestWpCli implements ServiceCliInterface', $customCommand);
```
Comment (dingo-d) [inline tests/WpCli/WpCliTest.php:43]: Expectations API 🙏🏼

**Review by dingo-d — CHANGES_REQUESTED** [review]
Left some minor comments. Otherwise looks good to me 👍🏼 

Can you just see about the service instantiation - test that part with automated tests?

**Review by dadadavorin — APPROVED** [review]

**Review by volfkarlo — APPROVED** [review]
👏

File: tests/WpCli/WpCliTest.php (hunk 2 above)
Comment (iruzevic) [reply tests/WpCli/WpCliTest.php:43]: 👍

**Pull request merged**